src/components/category-card/CategoryCard.styles.js

```diff
@@ -0,0 +1,30 @@
+export const styles = {
+  card: {
+    boxSizing: 'border-box',
+    maxWidth: '360px',
+    width: '100%',
+    padding: { xs: '20px 30px', lg: '25px 32px' }
+  },
+  imgTitleDescription: {
+    titleWithDescription: {
+      title: {
+        fontSize: '20px',
+        color: 'primary.900',
+        typography: 'h6',
+        mb: '4px'
+      },
+      description: {
+        fontSize: '14px',
+        color: 'primary.500',
+        typography: 'subtitle2',
+        fontWeight: 400
+      }
+    },
+    root: {
```

Comment:
Where do we use `root` styles?

Reply:
I pass it to the ImgTitleDescription component

![image](https://github.com/Space2Study-UA-1156/Client-01/assets/32570823/789e856e-2592-47e0-89cb-b7400b402ac4)